templates/jvb-statefulset.yaml

```diff
@@ -125,6 +133,12 @@ spec:
         ports:
         - containerPort: 9090
           name: websocket
+        - containerPort: {{ $.Values.jvb.nodeportPrefix }}{{ add $shard 3 }}00
```

Comment:
I am not sure this will work. Pod will listen on 30300 all the time, the NodePort Service will make the job to route from 303XX (XX depending of JVB index) to 30300, however the JVB will think he is on 30300. And if I change that with an env var then it will actually listen on 303XX and will still fail.

Reply:
Right. This was only coincidentally because I wasn't going above a single JVB